templates/commands/render_action.go

```diff
@@ -142,7 +142,7 @@ var templateKeyErrRegex = regexp.MustCompile(`map has no entry for key "([^"]*)"
 // there's no reason to print out spec file location in an error message. If
 // template execution fails because of a missing input variable, the error will
 // be wrapped in a unknownTemplateKeyError.
-func parseAndExecuteGoTmpl(pos *model.ConfigPos, tmpl string, inputs map[string]string) (string, error) {
+func parseAndExecuteGoTmpl[T any](pos *model.ConfigPos, tmpl string, inputs map[string]T) (string, error) {
```

Comment:
What if we just make inputs `map[string]any`?

Reply:
I don't think the type system will allow that (passing a map[string]string as a map[string]any). Feel free to send me a patch if you can get it to work.